index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './test/test-kit';
+export { getAbiByContractRegistryKey } from './abi-loader';
```

Comment:
Since you moved the main path to this index.js file - consider adding a comment in test-kit.js stating that it is deprecated and nothing should be added there. or - just copy paste everything here instead and remove that file

Reply:
I think that in the future the testkit will be a separate entry point to this package (because many ABI consumers don't need and sometimes even can't use the testkit).